esp32/modules/appglue.py

```diff
@@ -21,7 +21,8 @@ def start_ota():
     esp.rtcmem_write(1,254)
     deepsleep.reboot()
 
-def start_bpp():
+def start_bpp(duration):
```

Comment:
So nothing is done with this argument currently?
Let's at least save it in the RTC mem